operator/src/main/resources/scripts/startServer.sh

```diff
@@ -162,6 +162,24 @@ function copySitCfg() {
   fi
 }
 
+# Returns the count of the number of files in the specified directory
+function countFilesInDir() {
+  dir=${1}
+  cnt=`find ${dir} -type f | wc -l`
```

Comment:
I think the $? test below will always return success since 'wc -l' succeeds even though the find failed.   To counteract, set 'pipefail'.   I think you can set it for the entire script at the beginning of the script and it will cover functions as well (but I haven't tested to confirm), but if you want to localize it to this function then do this:
```
function countFilesInDir() {
   (
   set -o pipefail
   dir=${1}
   cnt=`find ${dir} -type f | wc -l`
   [ $? -ne 0 ] && trace SEVERE "failed determining number of files in '${dir}'" && exitOrLoop
   trace "file count in directory '${dir}': ${cnt}"
   return ${cnt}
   )
 }
```